package/mainflux/mainflux.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-MAINFLUX_VERSION = master
+MAINFLUX_VERSION = 0.10.0
```

Comment:
@netoax, don't we need [Mainflux 0.11.0](https://github.com/mainflux/mainflux/releases/tag/v0.11.0), for the [app token (Mainflux's API Key) duration to be configurable to `forever`](https://github.com/CESARBR/knot-cloud-sdk-js-authenticator/pull/17#event-3429943561)? Here's the fixed a [bug](https://github.com/mainflux/mainflux/issues/1103) in 0.11.0.